docs/data/base/components/snackbar/snackbar.md

```diff
@@ -0,0 +1,133 @@
+---
+product: base
+title: Unstyled React Snackbar component and hook
+components: SnackbarUnstyled
+githubLabel: 'component: snackbar'
+---
+
+# Unstyled snackbar
+
+<p class="description">The SnackbarUnstyled component informs users that an action has been or will be performed by the app.</p>
+
+## Introduction
+
+A snackbar provides users with a brief message about app processes without interrupting their activity or experience.
+
+The `SnackbarUnstyled` component is built to appear temporarily in the corner of the screen to inform users of an action that the app is taking.
```

Comment:
Oh yeah, I forgot about that! I really like the demo for this in the Material UI docs. Do you think it would be worthwhile to recreate it with Base? (I do 😁)

Reply:
I think we should not recreate it with Base since there is no `anchorOrigin` prop and a unstyled Snackbar can actually be placed anywhere on the screen.

I just wanted to point out the placement convention of a Snackbar component that it can be aligned on center as well in applications, not just on corners.